canonical-web-and-design/canonical.com
# Add careers filters to the url

## Done
- Change the url when filter is changed on `/careers/{department}#available-roles`
- Add empty state message if no jobs match filter selection


## QA
- Run the website locally using `./run`
- Go to [/careers/sales#available-roles](http://0.0.0.0:8002/careers/sales#available-roles) and change the filter on the page. See that the url is changed when you do that.
- Make sure there is an empty state message if no matches for your filter.


## Issue
Fixes https://github.com/canonical-web-and-design/canonical.com/issues/43
